message_definitions/v1.0/ras_a.xml

```diff
@@ -0,0 +1,124 @@
+        This can be either set by an operator (through MAV_CMD_SET_POI_FOCUS) or through onboard software or external processes.
+        The receiving system should track the POI being reported based on its UID + time of detection + time of update.
+        Detection time happens once for the same UID, while time of update changes when a specific metadata of that POI gets changed (e.g. position).
+        The time of update should be changed on the sending system, based on the determined data in regards to that specific POI.
+        So, POIs that are received again should be updated if the time of update has changed.
+        Note: The sending system should repeat the current POIs at a fixed default rate of 2Hz to keep the protocol stateless). The fixed rate though can be set by the receiver using MAV_CMD_SET_MESSAGE_INTERVAL, which is advised for POIs that are not static or for which the state updates are high - decision on the rates of some specific POIs is at the implementers consideration.
+      </description>
+      <field type="uint32_t" name="time_boot_ms" units="ms">Timestamp (time since system boot).</field>
+      <field type="uint64_t" name="time_utc_detected" units="ms" invalid="0">Timestamp (time since UNIX epoch) of the POI detection, in UTC. 0 for unknown.</field>
+      <field type="uint64_t" name="time_utc_updated" units="ms" invalid="0">Timestamp (time since UNIX epoch) of the last POI update, in UTC. 0 for unknown.</field>
+      <field type="uint8_t" name="confidence_overall" units="%" invalid="UINT8_MAX">Generic confidence level. 0..100, UINT8_MAX when unknown.</field>
+      <field type="uint8_t" name="confidence_detection" units="%" invalid="UINT8_MAX">Confidence level of the POI detection. 0..100, UINT8_MAX when unknown.</field>
+      <field type="uint8_t" name="confidence_classification" units="%" invalid="UINT8_MAX">Confidence level of the POI classification. 0..100, UINT8_MAX when unknown.</field>
+      <field type="uint8_t" name="confidence_localization" units="%" invalid="UINT8_MAX">Confidence level of the POI localization. 0..100, UINT8_MAX when unknown.</field>
+      <field type="uint16_t" name="ttl" units="s">Time to live: If this time has elapsed, the POI should be deleted on the receiver side. A value of 0 should indicate no timeout.</field>
+      <field type="uint8_t" name="flags" display="bitmask">Bitmask for POI state. Bit 1: POI is in focus on camera, Bit 8: POI has been cleared and should be deleted</field>
+      <field type="uint64_t" name="uid" invalid="0">Unique ID for a given POI. Updates to a POIs information should use the same uid. 0 means unknown.</field>
+      <field type="int32_t" name="latitude" units="degE7" invalid="0">Latitude (WGS84) of the POI. If unknown: 0 (both Lat/Lon).</field>
+      <field type="int32_t" name="longitude" units="degE7" invalid="0">Longitude (WGS84) of the POI. If unknown: 0 (both Lat/Lon).</field>
```

Comment:
Latitude and longitude 0 is technically valid, so invalid should be something like INT32_MAX.

Reply:
Yeah you are right. Technically possible, but unlikely though. Will change to `INT32_MAX`.